cocos2d/core/camera/CCCamera.js

```diff
@@ -470,14 +470,21 @@ let Camera = cc.Class({
          * @static
          */
         findCamera (node) {
+            let camera=null;
             for (let i = 0, l = _cameras.length; i < l; i++) {
-                let camera = _cameras[i];
+                 camera = _cameras[i];
                 if (camera.containsNode(node)) {
                     return camera;
                 }
             }
-
-            return null;
+                if(!camera){
+                    if(_cameras.length>0) camera=_cameras[0]; 
+                    /// return the first camera instead of returning null. EditBox raises error if no camera found , 
+                        ///this may also prevent some other bugs in some other cases. Generally, gamedev would not want to check if the camera is null.. 
```

Comment:
Yes, return the main camera is not bad. But what if the real camera is activated after calling findCamera? It will cause a wrong rendering effect.
So if we still return null, everyone will try to relocate the node until a real camera is on.

Reply:
i think you did not get the point, please read my other answer , i need to know if the wontfix label will stay or if issue will be solved. and better cocos team may wait for respond of author, and solve ambiguouties , to not get developers stuck , before labeling "wontfix" or like ...

Reply:
The wontfix label means will solve the issue in a more strict way. I really appreciate your feedback, helping us to polish API usages.
I think I got the point. I will wait for your respond before we close this PR.